pyat/at/lattice/elements.py

```diff
@@ -991,58 +991,58 @@ class SimpleQuantDiff(_DictLongtMotion, Element):
     _BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES
     default_pass = {False: 'IdentityPass', True: 'SimpleQuantDiffPass'}
 
-    def __init__(self, family_name: str, beta_x: Optional[float]=1.0,
-                 beta_y: Optional[float]=1.0, emit_x: Optional[float]=0.0,
-                 emit_y: Optional[float]=0.0, sigma_dp: Optional[float]=0.0,
-                 tau_x: Optional[float]=0.0, tau_y: Optional[float]=0.0,
-                 tau_z: Optional[float]=0.0, U0: Optional[float]=0.0,
+    def __init__(self, family_name: str, betax: Optional[float]=1.0,
+                 betay: Optional[float]=1.0, emitx: Optional[float]=0.0,
+                 emity: Optional[float]=0.0, espread: Optional[float]=0.0,
+                 taux: Optional[float]=0.0, tauy: Optional[float]=0.0,
+                 tauz: Optional[float]=0.0, U0: Optional[float]=0.0,
```

Comment:
For PEP8 compliance, it should be:
```python
    def __init__(self, family_name: str, betax: float = 1.0,
                 betay: float = 1.0, emitx: float = 0.0,
                 emity: float = 0.0, espread: float = 0.0,
                 taux: float = 0.0, tauy: float = 0.0,
                 tauz: float = 0.0, U0: float = 0.0,
```
No `Optional` (it's never `None`, always float), and white space around the `=` sign.

Reply:
This is taken into account. Also the same thing in simple_ring. The Optional was not necessary